.circleci/config.yml

```diff
@@ -151,85 +151,70 @@ workflows:
       - build
       - hold:
           type: approval
+      - master:
+          requires:
+            - build
           filters:
             branches:
-              ignore: master
+              only: master
+      - pr:
           requires:
-          - build
+            - build
+            - hold
+          filters:
+            branches:
+              ignore: master
 
 
 
-      - e2eTestCurPRClusterState:
-          requires:
-          - hold
-      - e2eTestWIPPRClusterState:
-          requires:
-          - hold
       - e2eTestCurMasterClusterState:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
       - e2eTestWIPMasterClusterState:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
-
-
+            - master
+      - e2eTestCurPRClusterState:
+          requires:
+            - pr
+      - e2eTestWIPPRClusterState:
+          requires:
+            - pr
 
 
 
-      - e2eTestCurPRDraining:
-          requires:
-          - hold
-      - e2eTestWIPPRDraining:
-          requires:
-          - hold
       - e2eTestCurMasterDraining:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
       - e2eTestWIPMasterDraining:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
+      - e2eTestCurPRDraining:
+          requires:
+            - pr
+      - e2eTestWIPPRDraining:
+          requires:
+            - pr
 
 
 
-      - e2eTestCurPRScaling:
-          requires:
-          - hold
-      - e2eTestWIPPRScaling:
-          requires:
-          - hold
       - e2eTestCurMasterScaling:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
       - e2eTestWIPMasterScaling:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
+      - e2eTestCurPRScaling:
+          requires:
+            - pr
+      - e2eTestWIPPRScaling:
+          requires:
+            - pr
 
 
 
       - e2eTestCurMasterUpdate:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+          - master
       - e2eTestCurPRUpdate:
           requires:
-          - hold
+          - pr
```

Comment:
This is

```yaml
workflows:
  version: 2
  build_e2e:
    jobs:
      - build
      - hold:
          type: approval
      - master:
          requires:
            - build
          filters:
            branches:
              only: master
      - pr:
          requires:
            - build
            - hold
          filters:
            branches:
              ignore: master



      - e2eTestCurMasterClusterState:
          requires:
            - master
      - e2eTestWIPMasterClusterState:
          requires:
            - master
      - e2eTestCurPRClusterState:
          requires:
            - pr
      - e2eTestWIPPRClusterState:
          requires:
            - pr



      - e2eTestCurMasterDraining:
          requires:
            - master
      - e2eTestWIPMasterDraining:
          requires:
            - master
      - e2eTestCurPRDraining:
          requires:
            - pr
      - e2eTestWIPPRDraining:
          requires:
            - pr



      - e2eTestCurMasterScaling:
          requires:
            - master
      - e2eTestWIPMasterScaling:
          requires:
            - master
      - e2eTestCurPRScaling:
          requires:
            - pr
      - e2eTestWIPPRScaling:
          requires:
            - pr



      - e2eTestCurMasterUpdate:
          requires:
          - master
      - e2eTestCurPRUpdate:
          requires:
          - pr
```

Reply:
<img width="780" alt="screen shot 2018-10-25 at 12 50 38" src="https://user-images.githubusercontent.com/4608994/47498324-ce03ad00-d854-11e8-84a3-cf719e5f3e7d.png">

🎉

Reply:
You will need to define `pr` or `master` jobs here as well: https://github.com/giantswarm/aws-operator/pull/1221/files#diff-1d37e48f9ceff6d8030570cd36286a61R25

Reply:
Works also for master. As you can see hold is not green and tests are running.

<img width="736" alt="screen shot 2018-10-25 at 17 30 07" src="https://user-images.githubusercontent.com/4608994/47515824-bf7cbc00-d87b-11e8-87c1-6d4fb7024a99.png">